validate_test.go

```diff
@@ -55,6 +55,19 @@ func Test_validateFlags_GCE_ProjectID(t *testing.T) {
 	}
 }
 
+func Test_validateFlags_Azure_SubscriptionID(t *testing.T) {
```

Comment:
There should be a positive and negative test to prove that the code works. Testing only the negative case is 50% of what we need for this. Thanks for adding it.